src/components/search-bar/StudentInfo.js

```diff
@@ -1,62 +1,61 @@
 import { Button } from 'antd'
-import React from "react";
-import Modal from "./Modal.css"
-
+import React from "react"
+import '../../less/Modal.less'
```

Comment:
Changing the style import from the CSS file to LESS file is the only actual change in this file; everything else is just automagic whitespace changes by the linter.